public/usage-examples/geometry/point_point_distance/point_point_distance-1-simple-oop.cs

```diff
@@ -0,0 +1,37 @@
+﻿using SplashKitSDK;
+
+namespace PointPointDistance
+{
+    public class Program
+    {
+        public static void Main()
+        {
+            // Create and initialise new window with title and dimensions
+            Window wnd = SplashKit.OpenWindow("Distance From Center", 600, 600);
+            SplashKit.ClearScreen();
+
+            // Create circle at the center of window 
+            SplashKit.FillCircleOnWindow(wnd, Color.Red, 300, 300, 6);
+            SplashKit.RefreshScreen();
+            
+            // While window is open 
+            while(!SplashKit.QuitRequested())
+            {
+                SplashKit.ProcessEvents();
+
+                // Point of center 
+                Point2D center = SplashKit.ScreenCenter();
+
+                // Point of cursor position 
+                Point2D mouse = SplashKit.MousePosition();
+
+                // Print distance to terminal 
+                SplashKit.WriteLine(SplashKit.PointPointDistance(center, mouse));

```

Comment:
I've just reviewed this code and I have a suggested change.
After the print distance to terminal add a slight delay, I tested with `SplashKit.Delay(100)`
Otherwise you very quickly get thousands of lines printed to the terminal and it becomes a little pointless and hard to track.